### DISCUSSION TITLE
How to disable item faction/race/class restrictions?

### DISCUSSION THREAD

**Discussion by nordfold** [azerothcore/azerothcore-wotlk#16995] [Guides & Tips]
Good day to everyone!

When I try to put on [Silvermoon city tabard] on human character, the game gives me a "You can never use that item" error. Can you please tell me how I can disable such faction, race and class restrictions on items?

Unfortunately i didn't find any useful parameter in «item_template» table

**Comment by PossiblyNotTaken** [comment]
A bit late at this point but it's the FlagsExtra column that you'd have to change to 0.

UPDATE acore_world.item_template
SET FlagsExtra = 0
WHERE name LIKE "Silvermoon City Tabard";

**Reply by nordfold** [reply]
Thank you so much <3
